Project-II-Calculator/calculator/src/components/CalculatorComponent/MainButton.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import ActionButton from '../ButtonComponents/ActionButton'
+import NumberButton from '../ButtonComponents/NumberButton';
+const buttonStyle = {
+    color: 'black',
+    width: '25%',
+    height: '40px',
+    fontSize: '30px'
+}
+
+const actionStyle = {
+    color: 'white',
+    backgroundColor: 'red',
+    width: '25%',
+    height: '40px',
+    fontSize: '30px'
+}
+const MainButton = ({content})=>{
+    return (
+        <div className='top-button-section'>
+            <NumberButton style={buttonStyle} number={content.num1}/>
```

Comment:
This is my personal view on it, but I don't really like how you decided to break your components. 
To me would have made much more sense to have a components for numbers and another one for the operations.